src/skewed.erl

```diff
@@ -0,0 +1,153 @@
+-module(skewed).
+
```

Comment:
Could you add module documentation explaining with a skewed tree actually is? And how it relates to merkle trees?